### PR TITLE
Policies/testing

### LINKED ISSUE #39: Cypress testing for Policies page

### PR DESCRIPTION
**Pull request by catlynbowles**
Resolves #39 

** Describe the functionality to be tested: **
- website is fully tabbable
- compatible with screen reader
- mobile / responsive layout
- completes cypress testing for policies view

** Giphy of how you feel submitting this PR: **

### CODE REVIEW

File: cypress/e2e/statepolicies.cy.js [hunk 1]
```diff
@@ -1,58 +1,125 @@
...
+import legalpolicies from '../fixtures/legalpolicies.json';
+import illegalpolicies from '../fixtures/illegalpolicies.json';
+import atriskpolicies from '../fixtures/atriskpolicies.json';
+
+describe('State Policies View', () => {
+  beforeEach(() => {
+    cy.visit('http://localhost:3000/states-policies');
+    cy.interceptGQL(
+      'https://roed-to-aide-be.herokuapp.com/graphql?api_key=ca912ed1df0d1c0f014ec94e3c731881',
+      'GetPolicies',
```
Comment (lkarpins) [inline cypress/e2e/statepolicies.cy.js:11]: E U R E K A!!

**Review by lkarpins — APPROVED** [review]
GraphQL Testing was a doooozy! Awesome work, Cat! 🥳

**Review by Rallen13 — APPROVED** [review]
You crushed that testing! I'm gonna rip off your error handling test once this gets merged in!

File: cypress/e2e/statepolicies.cy.js [hunk 2]
```diff
@@ -1,58 +1,125 @@
...
\ No newline at end of file
+import policies from '../fixtures/policies.json';
+import legalpolicies from '../fixtures/legalpolicies.json';
+import illegalpolicies from '../fixtures/illegalpolicies.json';
+import atriskpolicies from '../fixtures/atriskpolicies.json';
+
+describe('State Policies View', () => {
+  beforeEach(() => {
+    cy.visit('http://localhost:3000/states-policies');
+    cy.interceptGQL(
```
Comment (irmakerdem) [inline cypress/e2e/statepolicies.cy.js:9]: gql specific interception! 🏈

File: cypress/e2e/statepolicies.cy.js [hunk 3]
```diff
@@ -1,58 +1,125 @@
...
+  it('Should display policys cards in alphabetical order with the state name and legal description', () => {
+    cy.dataCy('info-strip').children().first().contains('Alabama')
+    cy.dataCy('info-strip').children().next().contains('Abortion is completely banned in Alabama')
+
+    cy.dataCy('info-strip').children().last().contains('Arizona')
+    cy.dataCy('info-strip').children().next().contains('Yes. Abortion is legal in Arizona.')
+  })
+
+  it('Should should allow the user to sort by legal state status', () => {
+    cy.dataCy('legal').click({ timeout: 10000 })
```
Comment (irmakerdem) [inline cypress/e2e/statepolicies.cy.js:58]: why such a long 10s timeout?

File: src/Views/StatePoliciesView/StatePoliciesView.js [hunk 4]
```diff
@@ -82,33 +84,35 @@ const StatePoliciesView = () => {
...
-            className="policy-img"
-            src={fists}
-            alt="empowered fists"
-            data-cy="fists-img"
-            tabIndex='0'
-          ></img>
-        </div>
-        {error ? <p>error</p> :
+        }
+        {loading ? <Loading /> :
```
Comment (irmakerdem) [inline src/Views/StatePoliciesView/StatePoliciesView.js:115]: YES the loading component! 📈

**Review by irmakerdem — APPROVED** [review]

**Pull request merged**